psl/psl-core/src/builtin_connectors/postgres_datamodel_connector.rs

```diff
@@ -318,7 +318,7 @@ impl Connector for PostgresDatamodelConnector {
             DoublePrecision => ScalarType::Float,
             // Decimal
             Decimal(_) => ScalarType::Decimal,
-            Money => ScalarType::Float,
+            Money => ScalarType::Decimal,
```

Comment:
The mapping was incorrect. I spotted it as we're using this function for native type tests